CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ NOTE:
 
 ## Table of contents
 
-- [v1.1.0-rc.1](#v110---rc1)
+- [v1.1.0-r2.1](#v110---r21)
```

Comment:
The release is just "r2.1", if you like with rc as postfix to indicate that it is a pre-release (you could have copied that from r1.1)

```suggestion
- [r2.1 - rc](r21---rc)
```